### ISSUE TITLE
kinit: not found in docker

### ISSUE THREAD

**Issue by roelal** [edenhill/kafkacat#267]
I'm trying to connect to a kafka server secured with kerberos. When using the edenhill/kafkacat container, I get

`sh: kinit: not found`

followed by

`%3|1600348764.142|SASLREFRESH|rdkafka#consumer-1| [thrd:app]: Kerberos ticket refresh failed: kinit -R -t "/keytab" -k me@server || kinit -t "/keytab" -k me@server: exited with code 127`

Which leads me to believe that kinit is not installed in the container.

**Comment by edenhill** [comment]
Yeah, you will need to make a derived container with the kerberos tools installed, we can't add it to the image because it requires the default realm to be configured, and we don't know your realm.

**Comment by roelal** [comment]
Ah. I think, for debian, this will not ask for Kerberos config during install:
```
export DEBIAN_FRONTEND=noninteractive # already set in Dockerfile.debian
apt-get install krb5-user
```
(For alpine `apk`, I don't know if there's a way to disable interactive config.)

One could then mount his own `/etc/krb5.config` file defining the default realm, if needing to use kerberos. The other 99% of the users is not bothered, other than by a slightly bigger image ;)
